### PR TITLE
Update manual test case "node disconnection"

### PR DESCRIPTION
**Pull request by chanow816**
Update out of date test steps in case 2

### CODE REVIEW

**Review by khushboo-rancher — DISMISSED** [review]
LGTM

**Comment by khushboo-rancher** [comment]
@chanow816 Can you please add signoff. `git commit --amend --signoff`

**Review by kaxing — DISMISSED** [review]
Maybe we can also include the link to the test script,
> https://github.com/longhorn/longhorn/files/4864127/network_down.sh.zip

I suppose the GitHub file hosting is semi-permanent.

**Comment by chanow816** [comment]
Added --signoff in commit command and added disconnect script link in file content 

@khushboo-rancher , @kaxing

File: docs/content/manual/pre-release/node/node-disconnection.md [hunk 1]
```diff
@@ -11,11 +13,11 @@ https://github.com/longhorn/longhorn/issues/1545
 
 ### Case 2:
 1. Launch Longhorn.
-2. Launch a pod with the volume and write some data. (Remember to set liveness probe for the volume mount point.)
+2. Use statefulset launch a pod with the volume and write some data.
 3. Disconnect the node that the volume attached to for 100 seconds.
 4. Wait for the node back and the volume reattachment.
-5. Verify the data and the pod still works fine.
-6. Delete the pod and wait for the volume deleted/detached.
+5. During volume detached/attached, pod will auto delete/create
```
Comment (innobead) [inline docs/content/manual/pre-release/node/node-disconnection.md:19]:
```suggestion
5. After the volume is reattached, the pod will be automatically deleted and recreate.
```

**Review by innobead — APPROVED** [review]
LGTM, but the statement needs to modify a bit

**Review by innobead — APPROVED** [review]

**Pull request merged**